New package: CoordinateConventions v1.0.2

JuliaRegistrator opened the pull request:
- Registering package: CoordinateConventions
- Repository: https://github.com/ProjectTorreyPines/CoordinateConventions.jl
- Created by: @bclyons12
- Version: v1.0.2
- Commit: 290e6b7e062cebcb4b07d422d39718271ad042d0
- Reviewed by: @bclyons12
- Reference: https://github.com/ProjectTorreyPines/CoordinateConventions.jl/commit/290e6b7e062cebcb4b07d422d39718271ad042d0#commitcomment-144863849
- Description: COCOS transformations
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6ed553e8a53b464f547d9c7deb6bcc6d3590d5d4c443dc75cecd72df3bef21d8d8f6253cfc849f34eb44c7262b73929d2cc5927b63c46edd34ef29ea1e572aadb8a0822bfbaddaa152745db58b93016fa7bf20004712c23d4cf2ab535a259784299169d942eb1d2f46c981428bc6b95ecaedd9363b179c01e073f7013873ef61de90e4122e575b8387b1583f6655670e46aebef44f752ef366a4d38dee56cc4b7037598addc6fa9809cb746f6fba075bbff1261f717f7de1911ab7d93953c7790 -->

bclyons12 commented:
[noblock]

@aplavin It's an implementation of [COCOS - Coordinate Conventions](https://crppwww.epfl.ch/~sauter/cocos/) commonly used in fusion research. We will add this to the documentation.
Paper: [O. Sauter and S. Y Medvedev, "Tokamak Coordinate Conventions: COCOS", Comput. Phys. Commun. 184 (2013) 293](https://crppwww.epfl.ch/~sauter/cocos/Sauter_COCOS_Tokamak_Coordinate_Conventions.pdf)

cc: @orso82

orso82 commented:
Good point. I have updated the README.md and online documentation accordingly

JuliaRegistrator committed:
New package: CoordinateConventions v1.0.2

UUID: 7204ce3a-f536-43d2-be4a-fbed74e90d86
Repo: https://github.com/ProjectTorreyPines/CoordinateConventions.jl.git
Tree: 21526fff87d3552b6fe4230a7d541d9fcecd0317

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
